tutorials/algorithms/01_algorithms_introduction.ipynb

```diff
@@ -15,7 +15,7 @@
    "source": [
```

Comment:
<p>Just a small comment, I think it's probably better to decompose the circuit before drawing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1370/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>